sdk/ai/azure-ai-generative/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.0.0b2 (unreleased)
+- Fix LangChain importing path
```

Comment:
Can you add this to the 1.0.0b1, that version is till active.

Reply:
Yes I can, just a bit worry if people are using b1, their package won't get updated

Reply:
Discussed offline, will move back to b1